src/uu/dd/src/datastructures.rs

```diff
@@ -155,6 +157,12 @@ impl std::fmt::Display for InternalError {
 
 impl Error for InternalError {}
 
+impl UError for InternalError {
```

Comment:
If I remember correctly, the code of `1` is also the default implementation, so you could leave that out. However, it also makes sense to keep it, I guess.

Reply:
Thanks, I'll replace this with `impl UError for InternalError {}` instead and keep that in mind for next time.